### PR TITLE
[13.0][MIG]stock_push_delay

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Standard migration

@ForgeFlow

### CODE REVIEW

File: stock_push_delay/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockMove(models.Model):
```
Comment (rousseldenis) [inline stock_push_delay/models/stock.py]: Please split one model per file.

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@AaronHForgeFlow Please check pre-commit. development_status key should respect standard values.

**Comment by AaronHForgeFlow** [comment]
Trying to fix pre-commit here: https://github.com/OCA/stock-logistics-workflow/pull/727

**Comment by rousseldenis** [comment]
@AaronHForgeFlow Can you check pre-commit. Missing line

**Review by rousseldenis — APPROVED** [review]
Code review.

**Review by HviorForgeFlow — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
Sorry, I did not mean this PR, so please dismiss this message

**Comment by AaronHForgeFlow** [comment]
cc @LoisRForgeFlow

**Review by hparfr — APPROVED** [review]

**Pull request merged**